docker-compose.yml

```diff
@@ -6,14 +6,13 @@ services:
     restart: unless-stopped
     ports:
       - ${CORE_P2P_PORT:?err}:${CORE_P2P_PORT:?err} # P2P
+      - 127.0.0.1:${CORE_RPC_PORT:?err}:${CORE_RPC_PORT:?err} #RPC
     volumes:
-      - ./configs/${NETWORK:?err}/core/dashd.conf:/dash/.dashcore/dash.conf
+      - ~/.mn/${NETWORK:?err}/dashd.conf:/dash/.dashcore/dash.conf
```

Comment:
Shouldn't be a config name instead of network? Also, path to home dir is variable so we need to pass it here too.

Reply:
Set to config name for now. The home dir is currently not variable, but I can try to set it variable later.